src/components/Container/Container.scss

```diff
@@ -0,0 +1,117 @@
+@import './../../styles/fonts.scss';
+@import './../../styles/breakpoints.scss';
+@import './../../styles/alignment.scss';
+
+.container {
+  display: grid;
+  text-align: center;
+  max-width: $breakpoint-laptop-large;
+  margin: 0 16px;
+  padding: 5px;
+}
+
+// This column class could be removed entirely and the grid system would work. We could consiser adding the padding and margin to the layout classes, and controlling this in the layout classes.
+
+.column {
+  padding: 10px;
+  margin: 10px;
+}
+
+/* Layouts  */
+
+// 2 Columns
+
+.cols-2 {
+  grid-template-areas: 'column column';
```

Comment:
Not necessary to declare areas since its not a "layout"

Reply:
grid-template-columns should handle everything fine

Reply:
@mattlaguardia grid-template-areas removed.